B827EBFFFEB54B9E.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEB54B9E",
+    "servers": [
+      {
+        "server_address": "ttn-router-us-west",
```

Comment:
Are you sure this is the correct address? Unless you're doing something on the DNS side, this is not a recognized url for an instance of the router (check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses) 
Cheers